src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/alerts/AlertsServiceTest.kt

```diff
@@ -50,9 +50,7 @@ class AlertsServiceTest {
       fun setUp() {
         whenever(offenderBookingRepository.findAllByOffenderNomsId("A1234KT")).thenReturn(
           listOf(
-            booking(
-              listOf(),
-            ),
+            booking(),
           ),
```

Comment:
no longer pass in a list of alerts but `apply` so we can grab the booking and set on the alert.
That has a big impact on how the tests data is created unfortunately

Reply:
So the changes below are essentially more the duplicate alerts to a single booking and then a new test to show more recent bookings take priority over older bookings . But it is hard to see any of that in the diff